oxpy/bindings_includes/ConfigInfo.h

```diff
@@ -10,8 +10,8 @@
 
 #include <Utilities/ConfigInfo.h>
 
-#include "../pybind11/include/pybind11/detail/common.h"
-#include "../pybind11/include/pybind11/pybind11.h"
+#include <pybind11/detail/common.h>
+#include <pybind11/pybind11.h>
```

Comment:
These two lines can both be safely removed. In fact, I don't know why I put them there in the first place!